packages/client/components/ReactjiPicker.tsx

```diff
@@ -6,6 +6,7 @@ import {PALETTE} from '~/styles/paletteV3'
 import {MenuProps} from '../hooks/useMenu'
 import Menu from './Menu'
 import appleEmojis from 'emoji-mart/data/apple.json'
+import {getRecentArray, handleRecent} from '../utils/reactjiPickerUtils'
```

Comment:
nit (nit-pick, feel free to ignore): `...Utils` is quite generic. Since this is for handling recents logic only, a better name would be `utils/reactjiPickerRecents`